### ISSUE TITLE
`s` or `sec` in report opportunities

### ISSUE THREAD

**Issue by adamraine** [GoogleChrome/lighthouse#14337]
Moving this discussion from https://github.com/GoogleChrome/lighthouse/pull/14329#issuecomment-1230846765.

I could go either way but maybe `s` makes more sense to be consistent with older versions of LH.

**Comment by paulirish** [comment]
+1 for `s`.   sec totally throws me.

**Comment by paulirish** [comment]
CLDR over here being slackers

**Comment by brendankenny** [comment]
Of course, NIST is a branch of the US federal government, so we should maybe get other opinions before taking an american saying "no need to localize!" at face value :)

**Comment by adamraine** [comment]
`narrow` SGTM. I did a quick test to see what `narrow` looked like for `ms` and it only removed the space.

![Screen Shot 2022-08-31 at 1 55 06 PM](https://user-images.githubusercontent.com/6752989/187780323-11394746-3b67-41eb-9354-630e9f68b74c.png)

I'm thinking we keep `ms` as `short`
